github-dark.css

```diff
@@ -641,7 +641,7 @@
   .gist-quicksearch-result, .gist-quicksearch-result-group, .listgroup,
   .listgroup-item + .listgroup-item, .user-key-badge, .newsfeed-placeholder,
   .progress-bar-inline .progress-bar, .menu-heading, .border-gray-dark,
-  .discussion-item-review .blob-wrapper {
+  .discussion-item-review .blob-wrapper, .Box-header {
```

Comment:
You can remove this rule. The rule below has the same specificity (1 class, 1 !important) so the last rule will always win.